CHANGELOG.md

```diff
@@ -63,7 +64,7 @@ engine. [[GH-27620](https://github.com/hashicorp/vault/pull/27620)]
 * **Adaptive Overload Protection (enterprise)**: Enables Adaptive Overload Protection
 for write requests as a GA feature (enabled by default) for Integrated Storage.
 * **Audit Entry Exclusion**: Audit devices support excluding fields from entries being written to them, with expression-based rules (powered by go-bexpr) to determine when the specific fields are excluded.
-* **Workload Identity Federation UI for AWS (enterprise)**: Add WIF fields to AWS secrets engine. [[GH-28148](https://github.com/hashicorp/vault/pull/28148)]
+* **Feature Name**: Add WIF fields to AWS secrets engine. [[GH-28148](https://github.com/hashicorp/vault/pull/28148)]
```

Comment:
What happened with this one I wonder? The original line looks more correct.

Reply:
not sure? but can manually revert if we think that's the right thing to do.

Reply:
yeah not sure how it happened but agreed, I think just manually replacing that with the original text would be best